src/screens/MainScreen/styles.js

```diff
@@ -1,10 +1,10 @@
 import { StyleSheet } from 'react-native';
-import { BACKGROUND, YELLOW, LIGHT_YELLOW } from 'constants/colors';
+import { YELLOW, LIGHT_YELLOW } from 'constants/colors';
+import commonStyles from 'constants/commonStyles';
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    backgroundColor: BACKGROUND,
+    ...commonStyles.screenContainer,
```

Comment:
Just as a comment, this is the same as
```suggestion
    container: commonStyles.screenContainer,
```